kubernetes-client/src/main/java/io/fabric8/kubernetes/client/impl/ExtensionsAPIGroupClient.java

```diff
@@ -41,35 +42,60 @@
 
 public class ExtensionsAPIGroupClient extends ClientAdapter<ExtensionsAPIGroupClient> implements ExtensionsAPIGroupDSL {
 
+  /**
+   * @deprecated The extensions/v1beta1 API is deprecated. Use apps/v1 instead ({@link AppsAPIGroupDSL#daemonSets()
+   *             kubernetesClient.apps().daemonSets()}).
+   */
   @Override
   @Deprecated
   public MixedOperation<DaemonSet, DaemonSetList, Resource<DaemonSet>> daemonSets() {
```

Comment:
Do we also need to add these javadocs in ExtensionsAPIGroupDSL?

Reply:
arrg, they need to be there instead